goodtables/compat.py

```diff
@@ -43,13 +43,21 @@ def iterenc_utf8(data):
     from urllib import parse
     from urllib.request import urlopen
     from urllib.error import HTTPError
-    csv_reader = csv.reader
```

Comment:
As this change is only for Python 3 code, we are breaking Py2/3 compatibility. No?